### ISSUE TITLE
i trained my own data of  two classes(background & human) with the bisenet-resnet18-speed , and i have two questions:

### ISSUE THREAD

**Issue by zhangyunming** [ycszen/TorchSeg#60]
i trained my own data of  two classes(background & human) with the bisenet-resnet18-speed , and i have two questions:

1. when i train the model, only at epoch0 saved the weight,
"""Display Config"""
C.snapshot_iter = 50
the 50 means next saved weight will at the 50th epoch? how to saved the weight when a epoch stoped?

2. 
my config as:

"""Image Config"""
C.num_classes = 2  ##19,zym
C.background = 0  ### 
C.image_mean = np.array([0.485, 0.456, 0.406])  # 0.485, 0.456, 0.406
C.image_std = np.array([0.229, 0.224, 0.225])
C.target_size = 1024   ##不晓得原始不定
C.image_height = 384
C.image_width = 384
C.gt_down_sampling = 8#8
C.num_train_imgs = 127471###zym
C.num_eval_imgs = 1000

the C.gt_down_sampling = 8 will affect the train labels ? or just do it when val?
the C.target_size = 1024 , my data are not in same size , the target_size param is?

thank you 
a pytorch newer

**Comment by ycszen** [comment]
1. "snapshot_iter" means saving the checkpoint per snapshot_iter epochs.
If you want to save the model each epoch, you can set the number to 1. 

2. The gt_down_sampling will affect the training labels. You can change it for your own goal. 
The target size can not be used in this repo. You just need to change the height and width.
